apps/web/src/components/blocksuite/workspace-header/styles.css.ts

```diff
@@ -6,18 +6,9 @@ export const headerContainer = style({
   position: 'sticky',
   top: 0,
   background: 'var(--affine-background-primary-color)',
+  // @ts-ignore
```

Comment:
why?

Reply:
Not sure but without it ts will explain about it